webviz_ert/data_loader/__init__.py

```diff
@@ -150,6 +153,11 @@ def _query(self, query: str, **kwargs: Any) -> dict:
     def _get(
         self, url: str, headers: dict = None, params: dict = None
     ) -> requests.Response:
+        if headers is None:
+            headers = {"Token": self.token}
+        else:
+            headers.update({"Token": self.token})
+
         resp = _requests_get(f"{self.baseurl}/{url}", headers=headers, params=params)
```

Comment:
```suggestion
        if headers is None:
            headers = {}

        resp = _requests_get(f"{self.baseurl}/{url}", headers={**headers, "Token": self.token}, params=params)
```

1. Avoids code duplication
2. Avoids mutating input data